packages/nuxt/src/runtime/utils.ts

```diff
@@ -37,16 +35,6 @@ export function extractErrorContext(errorContext: CapturedErrorContext): Context
  * Exported only for testing
  */
 export function addSentryTracingMetaTags(head: NuxtRenderHTMLContext['head']): void {
-  const activeSpan = getActiveSpan();
-  const rootSpan = activeSpan ? getRootSpan(activeSpan) : undefined;
-
-  if (rootSpan) {
-    const traceParentData = spanToTraceHeader(rootSpan);
-    const dynamicSamplingContext = dynamicSamplingContextToSentryBaggageHeader(
-      getDynamicSamplingContextFromSpan(rootSpan),
-    );
-
-    head.push(`<meta name="sentry-trace" content="${traceParentData}"/>`);
-    head.push(`<meta name="baggage" content="${dynamicSamplingContext}"/>`);
-  }
+  const metaTags = getTraceMetaTags();
+  head.push(metaTags);
```

Comment:
l: Not sure if this is problematic but I see we did this differently before: If nuxt expects one tag per array item we should split the return value from `getTraceMetaTags()` at `\n` since we usually return two tags separated by a `\n`.

Reply:
I checked this and nuxt puts the html together in the correct way so this should be fine. But I would rather add a function parameter `{ options.asArray }` to return it as array instead of adding `\n` and splitting it again afterwards.

Reply:
I created a PR for this: https://github.com/getsentry/sentry-javascript/pull/13293

Reply:
As I closed the PR I just use the returned string now. This works with Nuxt as well and we don't have to split the string (slightly better performance).